.github/ISSUE_TEMPLATE/-maintainer--new-feature.md

```diff
@@ -0,0 +1,20 @@
+---
+name: "(Maintainer) New Feature"
+about: New feature description, filed by CRM maintainers
+title: ''
+labels: enhancement, source::maintainer
+assignees: ''
+
+---
+
+**What is the idea?**
+Describe what the feature is and the desired state.
+
+**Why is this needed?**
+Who would benefit from this feature? Why would this add value to them? What problem does this solve?
+
+**Alternatives that have been considered**
+A clear and concise description of any alternative solutions or features you've considered.
```

Comment:
I like the intent here but I don't think this is appropriate for this template.

I think of "alternatives" as something that is better suited for a design doc discussion. By the time we make a ticket, we're some-what comitted to executing on the work.